js/battle.js

```diff
@@ -3219,7 +3219,16 @@ var Battle = (function () {
 			if (kwargs.prepare || kwargs.anim === 'prepare') {
 				this.prepareMove(pokemon, move, target);
 			} else if (!kwargs.notarget) {
-				(kwargs.anim ? Tools.getMove(kwargs.anim) : move).anim(this, [pokemon.sprite, target.sprite]);
+				move = kwargs.anim ? Tools.getMove(kwargs.anim) : move;
+				if (target.side.active.length > 1 && (move.target === 'allAdjacent' || move.target === 'allAdjacentFoes')) {
```

Comment:
Basically, this should just be `if (kwargs.spread) {`

And then on the server we should be doing `attrLastMove('[spread]')`